sqldb/sqlc/queries/graph.sql

```diff
@@ -0,0 +1,123 @@
+/* ─────────────────────────────────────────────
+   features table queries
+   ─────────────────────────────────────────────
+*/
+
+-- name: CreateFeature :one
+INSERT INTO features (bit)
+VALUES ($1)
+ON CONFLICT (bit) DO UPDATE SET bit = EXCLUDED.bit
+RETURNING id;
+
+/* ─────────────────────────────────────────────
+   nodes table queries
+   ─────────────────────────────────────────────
+*/
+
+-- name: UpsertNode :one
+INSERT INTO nodes (
+    version, pub_key, alias, last_update, color, signature
+) VALUES (
+    $1, $2, $3, $4, $5, $6
+)
+ON CONFLICT (pub_key, version)
+DO UPDATE SET
+    alias = EXCLUDED.alias,
```

Comment:
Since these `ON CONFLICT ... DO UPDATE` statements are a bit hard to read sometimes, I made it a habit of adding a comment above what the expectation is. For example, the above on the feature bit is a no-OP because the bit field is part of the unique index.
But here we update the fields that aren't covered by the unique index.